adoc/deployment-preparation.adoc

```diff
@@ -8,42 +8,35 @@ and must be maintained on this machine, but it is not a member of the skuba clus
 [[ssh.configuration]]
 === Basic SSH Key Configuration
 
+*In order to successfully deploy {productname}, you need to have an SSH agent.* This is important, because it is required in order to use the installation tools `skuba` and `terraform`.
```

Comment:
```suggestion
*In order to successfully deploy {productname}, you need to have SSH keys loaded into an SSH agent.* This is important, because it is required in order to use the installation tools `skuba` and `terraform`.
```